### PR TITLE
make URLs into links (autolinking not supported)

### PR DESCRIPTION
**Pull request by pdurbin**
On the version at http://opensourcedesign.net/events/2017/10/open-source-design-summit-berlin the URLs aren't clickable.

### CODE REVIEW

File: _posts/2017-10-13-open-source-design-summit-berlin.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ We're organizing a summit for all designers and developers interested in cheerle
 
 October 13 is the arrival day with a social event in the evening. 14-15 are unconf days, 16-17 workshops/discussion, 18 is maybe brunch but also leaving day.
 
-Please see http://opensourcedesign.net/summit/ for more information and https://opencollective.com/opensourcedesign/events/opensourcedesign-summit to express interest in attending! You are also welcome to [join the conversation on GitHub](https://github.com/opensourcedesign/organization/issues?q=is%3Aissue+is%3Aopen+label%3Asummit).
+Please see <http://opensourcedesign.net/summit/> for more information and <https://opencollective.com/opensourcedesign/events/opensourcedesign-summit> to express interest in attending! You are also welcome to [join the conversation on GitHub](https://github.com/opensourcedesign/organization/issues?q=is%3Aissue+is%3Aopen+label%3Asummit).
```
Comment (evalica) [inline _posts/2017-10-13-open-source-design-summit-berlin.md:16]: I think the markdown syntax for links is "[]()".

Comment (jancborchardt) [reply _posts/2017-10-13-open-source-design-summit-berlin.md:16]: That’s correct syntax. :) `[Text](https://…)`

Comment (pdurbin) [reply _posts/2017-10-13-open-source-design-summit-berlin.md:16]: The `<` `>` syntax is correct too. Can we merge this? Nice seeing you on the call today! 😄

**Review by jancborchardt — APPROVED** [review]
Works! :)

**Pull request merged**